core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -338,6 +338,16 @@ private[spark] object UIUtils extends Logging {
       failed: Int,
       skipped: Int,
       total: Int): Seq[Node] = {
+    makeProgressBar(started, completed, failed, skipped, killed = 0, total)
+  }
+
+  def makeProgressBar(
```

Comment:
Can you change the main makeProgressBar method to always accept a killed argument (and then update the 2-3 other locations where this is called to pass in killed = 0)? It looks like the other places that use this method (e.g., the streaming BatchPage UI class) have access to the correct number of killed tasks, so should be setting it accordingly (and it's a little cumbersome to have to maintain both versions of this method).